tests/functional/ionIonBeam/ion_ion_beam1d.py

```diff
@@ -143,52 +151,85 @@ def growth_b_right_hand(run_path):
     # the factor "+1" is because we remove the DC component, so the value
     # given by argmax has also to miss this value
     omegas = np.fabs(np.fft.fft(damped_mode).real)
-    omega = 0.5*(omegas[1:omegas.size//2].argmax()+1)*2*np.pi/times[-1]
+    omega = 0.5*(omegas[1:omegas.size//2].argmax()+1)*2*np.pi/times[imax-1 -int(d/dt)]
 
-    print(omegas[0:6])
-    print(omegas[1:omegas.size//2].argmax())
-    print(2*np.pi/times[-1])
-    print(0.5*(omegas[1:omegas.size//2].argmax()+1)*2*np.pi/times[-1])
 
     return times, first_mode, popt[0], popt[1], damped_mode, omega
 
-
 def main():
-    from pybindlibs.cpp import mpi_rank
 
+    from pybindlibs.cpp import mpi_rank
+    d = 10 # The exponential fit is performed on [0, times[imax] - d]
```

Comment:
instead of `d`, could it be `time_offset`... and say why we have such an offset (I dunno)